stratum/algos/makefile

```diff
@@ -41,4 +41,4 @@ $(OUTPUT): $(OBJECTS)
 clean:
 	rm -f *.o
 	rm -f ar2/*.o
-	rm -f blake2/*.o	
```

Comment:
please restore this line

Reply:
Again I am sorry i was very tired when i did this. I have restored it